tensorboard/webapp/runs/store/runs_reducers_test.ts

```diff
@@ -1353,6 +1353,64 @@ describe('runs_reducers', () => {
 
       expect(nextState.data.initialGroupBy.key).toBe(GroupByKey.RUN);
     });
+
+    it('adds the experiment name column to the end of the runs table columns list', () => {
```

Comment:
It does add it to the end...not sure that is desired behavior. Maybe just name this 'adds the experiment name column to the runs table columns list'